misc/registrar_validation/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
```

Comment:
NB: .sh extension is redundant, so shouldn't be there.  You only should have that if it *needs* to be 'sourced" -- otherwise it could, e.g. be a binary or python or whatever. There is no need to know this is a script.